SGO/sgo/attributes/gender.ttl

```diff
@@ -0,0 +1,14 @@
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit:gender
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "gender";
+	dcterms:description """person's gender(sex). i.e male, female..""";
+	# For ranges, see http://dublincore.org/documents/dcmi-period/
```

Comment:
I don't think timing intervals are of much use here, I would remove that comment